### ISSUE TITLE
testing on Amazon Linux

### ISSUE THREAD

**Issue by clinty** [voxpupuli/puppet-selinux#232]
Successor to #230 

Is there anything that you'd like me to do with regards to https://app.vagrantup.com/mvbcoding/boxes/awslinux/ ?

**Comment by vinzent** [comment]
@clinty Back then when the dicussion about Amazon Linux was, I wasn't even able to enable SELinux with Amazon Linux manually (while running on EC2). Please provide steps howto enable selinux on Amazon Linux when running on EC2 (not some other VM).

**Comment by vinzent** [comment]
@oranenj @clinty I don't think it makes sense to run Amazon Linux tests on a local hypervisor like Virtualbox. As AmazonLinux is not a normal distro there are IMHO significant differences between AmazonLinux on a local hypervisor and AmazonLInux on EC2.

beaker tests can be run on EC2 directly you only need to have an account. There is also the so called free tier available for anybody to run these tests.

**Comment by clinty** [comment]
Off the top of my head, you need to do some subset of: install the relevant selinux policy packages, tweak the linux command line in /boot/grub/menu.lst, futz with /etc/selinux, and relabel the filesystem(s)

**Comment by clinty** [comment]
Not relevant to original "Amazon Linux", but https://aws.amazon.com/amazon-linux-2/faqs/

**Comment by Manouchehri** [comment]
> As AmazonLinux is not a normal distro there are IMHO significant differences between AmazonLinux on a local hypervisor and AmazonLInux on EC2.

@vinzent It's a somewhat "normal" KVM based hypervisor these days.
